edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -61,8 +61,8 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://image.freepik.com/vector-gratis/vector-onda-azul-transparente_1055-7084.jpg" width="70"
-                                    height="30" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
+                                    src="https://mylearningx.pearson.com/static/pearson-pols-theme/images/logo.png" width="40"
```

Comment:
Can we get this value from site configuration? @luismorenolopera

Reply:
Being an email, I can't use branding api, or do you mean something else?

Reply:
Yes, the problem here is the async task...